### ISSUE TITLE
Thanks for you amazing work

### ISSUE THREAD

**Issue by novageno** [unsky/RetinaNet#1]
seems some bugs still exist. Are you still under development?

**Comment by unsky** [comment]
yes, i was exploring the the paper to get the best performance , and if i have the results, i will update my code.

**Comment by Yc174** [comment]
Appreciate for your work. How is the test map?

**Comment by unsky** [comment]
@Yc174  test symbol is very to write, i will update the new version after making evaluation on voc dataset.
